src/app/services/auth.service.ts

```diff
@@ -137,7 +139,9 @@ export class AuthService {
  */
 export function getClientSettings(): UserManagerSettings {
   return {
-    authority: environment.identityServerUrl,
+    // authority: environment.identityServerUrl,
+    authority: "https://localhost:5005/",
```

Comment:
This value should be in the environment variables. Also remove the commented unused lines of code